CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Share links allow data attributes ([PR #3072](https://github.com/alphagov/govuk_publishing_components/pull/3072))
 * Update to LUX 304 ([PR #3070](https://github.com/alphagov/govuk_publishing_components/pull/3070))
+* Add `aria-controls` and `aria-describedby` attribute options to the button component
```

Comment:
Nitpick - can this match the PR name? (you can change either of them).

Also please add a link to the PR as shown in the other entries.